.github/workflows/labeler.yml

```diff
@@ -0,0 +1,25 @@
+name: Labeler
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2.3.3
+
+      - name: Run Labeler
+        if: success()
+        uses: crazy-max/ghaction-github-labeler@v3.1.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```

This is equivalent to the default value `{{ github.token }}`. See:

- https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret
- https://github.com/marketplace/actions/github-labeler#inputs